src/test/cli/ceph-conf/show-config-value.t

```diff
@@ -18,3 +18,18 @@
   mon_host=$public_network
   $mon_host
   $ rm $TESTDIR/ceph.conf
+
+Name option test to strip the PID
+=================================
+  $ echo '[client]' > $TESTDIR/ceph.conf
```

Comment:
nit, use here document for injecting settings. see `src/test/cli/ceph-conf/option.t`.

Reply:
Sure.